var/spack/repos/builtin/packages/openfoam-com/package.py

```diff
@@ -258,12 +258,14 @@ class OpenfoamCom(Package):
     """
 
     homepage = "http://www.openfoam.com/"
-    baseurl  = "https://sourceforge.net/projects/openfoamplus/files"
+    baseurl  = "https://sourceforge.net/projects/openfoamplus/files/"
+    gitrepo =  "https://develop.openfoam.com/Development/OpenFOAM-plus.git"
```

Comment:
Flake8 doesn't like this line:
```
var/spack/repos/builtin/packages/openfoam-com/package.py:262: [E222] multiple spaces after operator
```
If you want things to line up, add a space before the equals sign but make sure there is only one space after the equals sign.